stable/fairwinds-insights/templates/secret.yml

```diff
@@ -28,9 +28,9 @@ metadata:
     name: fwinsights-timescale
 type: Opaque
 stringData:
-    PATRONI_SUPERUSER_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password | b64enc }}
-    PATRONI_REPLICATION_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password | b64enc }}
-    PATRONI_admin_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password | b64enc }}
+    PATRONI_SUPERUSER_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password }}
+    PATRONI_REPLICATION_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password }}
+    PATRONI_admin_PASSWORD: {{ default (randAlphaNum 32) .Values.timescale.password }}
```

Comment:
🤔 you sure about this one? that seems surprising

Reply:
I noticed it was double encoding. Looks liek stringData already applies base 64 endoding